### ISSUE TITLE
searchfy.py --maltego doesn't disable banner

### ISSUE THREAD

**Issue by nealrauhauser** [i3visio/osrframework#169]
the --maltego flag should disable the banner, but it does not. Any transform using searchfy.py fails.

This is the invocation copied directly from the transform in Maltego 4

searchfy.py -p twitter --maltego -q  nrauhauser

---

 / _ \/ ___||  _ |  ___| __ __ _ _ __ ___   _____      _____  _ **| | __
| | | _** | |_) | |_ | '**/ _` | '_ ` _ \ / _ \ \ /\ / / _ | '**| |/ /
| |_| |___) |  _ <|  _|| | | (_| | | | | | |  **/\ V  V / (_) | |  |   <
 \_**/|____/|_| \__|  |_|  __,_|_| |_| |_|_**| _/_/ _**/|_|  |_|_\ 

```
            Version:      OSRFramework 0.11.5
            Created by:   Felix Brezo and Yaiza Rubio, (i3visio)
```

<MaltegoMessage>

**Comment by nealrauhauser** [comment]
Lack of conditional at line 74 in searchfy.py, this fixes it

#NRR mod 2016-09-08
    if not args.maltego:
        print banner.text
        sayingHello = """searchfy.py Copyright (C) F. Brezo and Y. Rubio (i3visio) 2016

**Comment by i3visio** [comment]
Confirmed  bug even in recent versions (current is 0.13.2) and beta versions (0.14.0b1). `sudo pip install osrframework --upgrade` would be enough to upgrade to the most recent stable version. We'll try to fix then for the 0.14.0 release which is coming soon... ;) Anyway, thanks for the report!

**Comment by i3visio** [comment]
Prepared to be released in 0.14.6.

**Comment by i3visio** [comment]
To be released in 0.15.0 on December 21st.
